app/zpt/handler/geos.py

```diff
@@ -0,0 +1,53 @@
+import numpy as np
+
+
+def geoid_radius(latitude):
```

Comment:
It feels like a bad idea to implement this ourselves. Is this even allowed by Craig Haley's license whatever it is..?